experimental/BasisLieHighestWeight/src/TensorModels.jl

```diff
@@ -30,9 +31,9 @@ function tensor_matrices_of_operators(
     end
     wi = ZZ.(1:length(highest_weight) .== i) # i-th fundamental weight
     matrices_of_operators = [
-      _tensor_product(mat_temp, _tensor_power(mat_wi, highest_weight_i)) for
+      _tensor_product(mat_temp, _tensor_power(mat_wi, 1)) for
```

Comment:
```suggestion
      _tensor_product(mat_temp, mat_wi)) for
```
bei exponent 1 brauchst du gar keine potenz